libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -303,15 +302,34 @@ void AP_MotorsHeli_Single::calculate_roll_pitch_collective_factors()
         _collectiveFactor[CH_1] = 1;
         _collectiveFactor[CH_2] = 1;
         _collectiveFactor[CH_3] = 1;
+    }
```

Comment:
another formatting issue.  this brace should not be on a separate line as the "elseif {"
the line should look like this
} elseif (conditional statement) {